src/utils.ts

```diff
@@ -25,8 +25,13 @@ export function isBeautifierPackageName(packageName: string): boolean {
 
 export function loadBeautifiers(beautifierNames: string[]): Unibeautify {
   const beautifiers = beautifierNames.map(beautifierName => {
-    return requireg(beautifierName).default;
-  });
+    try {
+      return requireg(beautifierName).default;
+    } catch (error) {
+      console.error(error);
+      return null;
```

Comment:
@szeck87 Did this work?

Reply:
Yes it did, no more unhandled promise rejections.